torch/CMakeLists.txt

```diff
@@ -65,6 +66,7 @@ set(TORCH_PYTHON_INCLUDE_DIRECTORIES
     ${CMAKE_BINARY_DIR}/caffe2/aten/src
     ${CMAKE_BINARY_DIR}/third_party
     ${CMAKE_BINARY_DIR}/third_party/onnx
+    ${CMAKE_BINARY_DIR}/third_party/breakpad/src
```

Comment:
This is probably not needed (TORCH_PYTHON_INCLUDE_DIRECTORIES is public dependencies, but breakpad should be private)